vendor/etc/init/cashsvr.rc

```diff
@@ -1,6 +1,7 @@
 on post-fs-data
-    # create socket for cashsvr
+    # create socket and data directory for cashsvr
     mkdir /dev/socket/cashsvr 0755 system system
+    mkdir /data/vendor/cashsvr 0755 system system
```

Comment:
If this dir is owned by `system:system`, root will have to use `dac_override` to create a file in it
So to get around that weird `rw- --- ---` behaviour on `O_CREAT`, let's use `root root`